src/js/components/ColumnMenuItem.vue

```diff
@@ -6,40 +6,31 @@
 
 <script>
 
-import { mapGetters } from 'vuex'
-
 export default {
   props: {
     dataId: Number,
     name: String,
   },
-  data() {
-    return {
-    }
-  },
-  computed: {
-    ...mapGetters(['username', 'sessionID', 'userID'])
-  },
   methods: {
-    fetchChildren() {
+    // fetchChildren() {
 
-      // we'll lazy load children, but cache them
-      if (this.lastFetch !== this.requestType[this.menuType]) {
-        this.$post('resources/cgi-bin/GetImageData.pl', {
-        transaction: this.requestType[this.menuType],
-        combID: this.scrollDataID,
-        user: this.userID,
-        version_id: this.versionID,
-      })
-      .then(res => {
-        if (res.status === 200 && res.data) {
-          this.children = res.data.results
-          this.lastFetch = this.requestType[this.menuType]
-        }
-      })
-      .catch(console.error)
-      }
-    },
+    //   // we'll lazy load children, but cache them
```

Comment:
@Bronson-Brown-deVost : Can you give me a quick rundown of why this is commented out versus straight deleted?

Reply:
Yeah, it will be implemented in the near future, but I have to write (a lot of) server side code first and will also need to add some data to the database for that server side code to return any results.  Sorry, I should have made a comment within the code itself to that effect.

Reply:
Gotcha. Makes sense.